frontend/src/components/PageButton/ReservationPageButton.tsx

```diff
@@ -14,6 +16,9 @@ const ReservationPageButton = () => {
           <S.Icon />
         </S.IconWrapper>
         <p>예약</p>
+        {data && data.length !== 0 && (
+          <S.ReceivedReservationCount>{data.length}</S.ReceivedReservationCount>
+        )}
```

Comment:
1. query의 return type을 줘야할 것 같습니다. data가 any라서 코드파악이 어렵네용
2. 
```suggestion
        {data?.length !== 0 && (
          <S.ReceivedReservationCount>{data.length}</S.ReceivedReservationCount>
        )}
```
로 변경하는게 가독성이 좋아보이는데 어떻게 생각하시나요?